contribute-docs/style-guide/seo.md

```diff
@@ -0,0 +1,553 @@
+
+
+**Ensure accessibility**
+
+* Avoid relying solely on images to convey critical information; always provide a text alternative.  
+* For videos, include captions and, if possible, a transcript.
+
+**Maintain quality and consistency**
+
+* Use high-quality, clear images and consistent styling (e.g., borders, background).
+
+**Reference visuals in text**
+
+* Clearly reference images or videos in the body copy (e.g., "See Figure 1" or "Watch the setup video below").
+
+**Cite sources when needed**
+
+* If using third-party visuals, ensure you have permission and provide appropriate attribution.
+
+**Consider structured data**
+
+* Where applicable, use structured data (e.g., VideoObject, ImageObject) to help search engines understand and display multimedia in rich results.  
+* Check with the documentation team for existing capabilities or implementation guidance. If you're an external contributor, open a GitHub issue to discuss implementation options.
+
+## URLs
+
+A URL (Uniform Resource Locator) is the web address of a page. Well-structured URLs benefit both SEO and user experience.
+
+Having a URL that clearly reflects what the page is about, can increase click-through-rates within the search engine results page by matching the user query. The URL is also displayed to users in the top of a browser’s search bar. Allowing the URL to be clear, concise, and easy to read, supports a positive on-site experience.
+
+URLs should not have special characters to ensure readability. They should also be lower-case to prevent issues with multiple URL versions that can signal duplicate content to search engines.
+
+Best practices:
+**Reflect the page topic**
+
+* Use keywords that accurately describe the page’s subject.  
+* Align the URL with the page title and H1 heading for consistency.
+
+**Keep URLs short and readable**
+
+* Avoid unnecessary words or parameters.  
+* Use hyphens (-) to separate words for clarity.
+
+**Use lowercase letters**
+
+* Always use lowercase to prevent duplicate content issues (e.g., /Guide/ vs. /guide/).
+
+**Avoid special characters**
+
+* Stick to letters, numbers, and hyphens. Avoid spaces, underscores, or special symbols.
+
+Ensure the Hierarchy is Reflected in the URL Structure
+
+* Structure URLs to mirror the documentation’s content hierarchy, nesting child pages under their logical parent directories (e.g., /docs/solutions/security/ai/ai-assistant).  
+* Ensure each child page has a valid parent URL that leads to a parent page.
+
+## Meta keywords tag
+
+Meta keywords are not important for SEO on Google and Bing, as they have stated that they ignore this meta tag. However, some other search engines like Yandex may still use them as a low weight ranking factor.
+
+It’s not necessary to add, but if needed, keywords should be kept to 5 keywords, separated by commas, while avoiding repetition, and duplication.
```

Comment:
> It’s not necessary to add, but if needed, keywords should be kept to 5 keywords, separated by commas, while avoiding repetition, and duplication.

Do have a way to accomplish this in Docs V3? I haven't seen it mentioned in https://docs-v3-preview.elastic.dev/elastic/docs-builder/tree/main/syntax/frontmatter